src/core/properties/ordinalproperty.cpp

```diff
@@ -91,4 +91,14 @@ OrdinalPropertyState<vec3> util::ordinalColor(const vec3& value,
             PropertySemantics::Color};
 }
 
+OrdinalPropertyState<vec3> util::ordinalLight(const vec3& pos, float min, float max,
```

Comment:
isn't that even more generic than light? It's a position...

Reply:
it has the light semantics, but I was also considering adding a pos something... but did not have a need for it at the moment...